server/lib/scene/index.js

```diff
@@ -45,6 +47,7 @@ const SceneManager = function SceneManager(
   this.http = http;
   this.gateway = gateway;
   this.brain = brain;
+  this.service = service;
```

Comment:
I'm not sure we want to inject services into the "scene" instance.

I think we should keep the same flow as the current MQTT integration:

- Only modify the MQTT integration
- Add a listener in the MQTT integration, and then emit an event (to start a scene)

Reply:
You are right, I have modified my code to use `handleGladysMessage()`